### PR TITLE
Batch bug lookup

### PR DESCRIPTION
**Pull request by thegreyd**

### CODE REVIEW

File: elliottlib/cli/find_bugs_cli.py [hunk 1]
```diff
@@ -226,8 +225,13 @@ def find_bugs_cli(runtime: Runtime, advisory, default_advisory_type, mode, check
             sweep_cutoff_timestamp = bzutil.approximate_cutoff_timestamp(runtime.assembly_basis_event, brew_api, runtime.rpm_metas() + runtime.image_metas())
 
         if sweep_cutoff_timestamp:
-            green_print(f"Filtering bugs that have changed to one of the desired statuses before the cutoff time {datetime.utcfromtimestamp(sweep_cutoff_timestamp)}...")
-            qualified_bugs = bzutil.filter_bugs_by_cutoff_event(bzapi, bugs, status, sweep_cutoff_timestamp)
+            green_print(f"Filtering bugs that have changed ({len(bugs)}) to one of the desired statuses before the "
+                        f"cutoff time"
+                        f" {datetime.utcfromtimestamp(sweep_cutoff_timestamp)}...")
+            qualified_bugs = []
+            for chunk_of_bugs in chunk(bugs, 500):
```
Comment (locriandev) [inline elliottlib/cli/find_bugs_cli.py:232]: I always prefer explicit module level "constants" insted of hardcoded numeric values that will be hard to maintain in the future

File: elliottlib/util.py [hunk 2]
```diff
@@ -687,3 +687,8 @@ def go_suffix_for_arch(arch: str) -> str:
 def brew_suffix_for_arch(arch: str) -> str:
     arch = brew_arch_for_go_arch(arch)  # translate either incoming arch style
     return brew_arch_suffixes[brew_arches.index(arch)]
+
+
+def chunk(l, n):
```
Comment (locriandev) [inline elliottlib/util.py]: How about adding type hints to this function?

Comment (locriandev) [reply elliottlib/util.py]: I would also pick more meaningful names for parameters: as I understand, "l" will always be a bug list while "n" is the batch size

Comment (thegreyd) [reply elliottlib/util.py]: Good point, added

**Pull request merged**